javascriptv3/example_code/iam/src/iam_getaccountpasswordpolicy.js

```diff
@@ -0,0 +1,28 @@
+/* Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+ABOUT THIS NODE.JS EXAMPLE: This example works with AWS SDK for JavaScript version 3 (v3),
+which is available at https://github.com/aws/aws-sdk-js-v3. This example is in the 'AWS SDK for JavaScript v3 Developer Guide' at
+https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide/iam-examples-managing-users.html.
+
+Purpose:
+iam_getaccountpasswordpolicy.js demonstrates how to retrieves the password policy for the Amazon Web Services account.
```

Comment:
iam_getaccountpasswordpolicy.js demonstrates how to retrieve the password policy for the AWS account.

Reply:
done